src/earn/BeforeDepositBottomSheet.tsx

```diff
@@ -25,14 +25,14 @@ import { Spacing } from 'src/styles/styles'
 import { TokenBalance } from 'src/tokens/slice'
 import { getTokenAnalyticsProps } from 'src/tokens/utils'
 
-function ActionCard({ action }: { action: BeforeDepositAction }) {
+export function ActionCard({ action }: { action: BeforeDepositAction | WithdrawAction }) {
```

Comment:
Might be worth moving this into its own file. Maybe even share with other parts of the app (token details, etc), seems like it's something we use throughout the app and would be nice to change styles in a single place for mobile stack apps